tdvt/samples/connector-tests/matchesattributes.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>   
+    <testCase match='true'>
+        <!-- Does not normalize attributes first. -->
+        <connection authentication='username-password' class='postgres_odbc' dbname='TestV1' odbc-native-protocol='' one-time-sql='' port='5432' server='postgres' username='test' />
```

Comment:
For the moment anyway postgres_odbc/jdbc don't use 'authentication' attribute, remove.

Reply:
Removed